### PR TITLE
Fix #18837: Style option reimplemented in context menu

### LINKED ISSUE #18837: Re-implementation of the 'Style' option in context menu from MU3
### Your idea

style button nothing in MU4

There was a style option in right click context menu in MU3, but it was removed in MU4; yes I can still save and load styles, but this one is still helpful when creating a style or temporary changing something, or I'll have to dig through a long and random style list without search function to find the entry I need.
It _is_ a little confusing with the _properties_ tab, but still useful sometime, especially for text styles. maybe we can move everything in the style list to _properties_ tab? is this already in a todo list?

### Problem to be solved

See idea section above

### Prior art

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by mathesoncalum**
Resolves: #18837
Resolves: #19203

### CODE REVIEW

File: src/notation/view/widgets/editstyle.cpp [hunk 1]
```diff
@@ -86,6 +86,7 @@ static const QStringList ALL_PAGE_CODES {
     "staff-text",
     "tempo-text",
     "lyrics",
+    "expression",
```
Comment (cbjeukendrup) [inline src/notation/view/widgets/editstyle.cpp:89]: It looks like this might resolve #19203. That would be worth trying out!

Comment (mathesoncalum) [reply src/notation/view/widgets/editstyle.cpp:89]: Yeah it should do!

**Review by RomanPudashkin — APPROVED** [review]

**Comment by DmitryArefiev** [comment]
Found some things:

1. #19203 looks good, but it's working from Properties and **doesn't work** from **right click menu**
2. playback techniques such as **arco, pizz etc** don't lead to **Styles>Staff text** as they do in MS3 (but in MS3 they were a Staff text and in MS4 they are a Playing technique annotation) 
3. Title text leads to Style>Text, but **Subtitle, Composer** etc. **don't lead** to Style>Subtitle/Composer respectively


https://github.com/musescore/MuseScore/assets/90187801/615795a9-2a28-4ed6-be79-c6fa1097bded

**Comment by DmitryArefiev** [comment]
@mathesoncalum I can log it separately (all and some items)

**Comment by mathesoncalum** [comment]
Thanks @DmitryArefiev!

I've addressed a couple of your points in this latest commit. Instrument names should now take you to "Text Styles" and playing techniques will lead you to "Stave Text".

In terms of resolving selected elements to specific sub-sections of Text Styles, we don't currently have a method for doing this. This may indeed constitute a separate issue.

**Comment by DmitryArefiev** [comment]
@mathesoncalum Yes, sure. I'll log it separately. Thanks!

Tested #18837 #19203 on Win10, Mac13, LinuxUbuntu 22.04 - **Implemented/Fixed**

**Pull request merged**